README.md

```diff
@@ -144,8 +141,12 @@ The best way to connect to DiceDB is using DiceDB CLI and you can install it by
 pip install dicedb-cli
 ```
 
-> Because DiceDB speaks Redis dialect, you can connect to it with any Redis Client and SDK also.
-> But if you are planning to use the `QWATCH` feature then you need to use the DiceDB CLI.
+## Client Compatibility
+
+DiceDB is fully compatible with Redis protocol, allowing you to connect using any existing Redis client or SDK.
+
+> [!NOTE]
+> The `Q.WATCH` feature is only accessible through the DiceDB CLI.
```

Comment:
Not correct, can be accessed by `SUBSCRIBE` command via redis-cli.

Reply:
We don't plan on supporting this long term so we may skip adding this in the official documentation.